### DISCUSSION TITLE
Artist name not shown on Sonos app while it's shown on MA interface

### DISCUSSION THREAD

**Discussion by miniworm** [music-assistant/support#2759] [Q&A] [answered]
I have latest server (2.2.0b6) runnig with my local library provided through Subsonic server (Navidrome) and Sonos player (Sonos Era 100). Playback is generally ok but on Sonos app I see only name of the song bu not artist name or album. This is while on MA UI I see all of this info provided. Below you can see screenshots from both interfaces.
![357783135-42e10147-9ca2-40a2-b135-e6f9ba239d0d](https://github.com/user-attachments/assets/f0734a13-edcf-45dc-b921-2895b15e5415)
![357783151-eb9f4f46-380f-4958-ba2a-7a47654ab5cf](https://github.com/user-attachments/assets/4f35d334-de29-495c-8d2a-c147eb43048a)

**Comment by miniworm** [comment]
Additionally things get mixed - as you can se Sonos sees no title ("Bez tytułu" means "No title") and title is shown as album (or artist?)

**Comment by miniworm** [comment]
The same file started from Sonos (it's local file in both cases) is properly shown in Sonos.
![Zrzut ekranu 2024-08-18 o 12 24 43 jpeg](https://github.com/user-attachments/assets/46e7d6e4-0642-4dd2-a1a9-b3cbdb313928)

**Comment by miniworm** [comment]
Also file started from Sonos is properly shown in both Sonos and MA:
![Zrzut ekranu 2024-08-18 o 12 26 56 jpeg](https://github.com/user-attachments/assets/60700f33-5c3a-43ee-9696-86061af42ef8)
![Zrzut ekranu 2024-08-18 o 12 27 25 jpeg](https://github.com/user-attachments/assets/22a2607b-498c-4f0f-b5e3-f56f4db67720)

**Reply by OzGav** [reply, marked as answer]
Looks like this question got missed. MA is not designed to interact with third party software. Anything that does work is a bonus. Anything that doesn’t work is to be expected.
